### ISSUE TITLE
Use `object-fit: cover` to avoid squashing/stretching profile photos

### ISSUE THREAD

**Issue by dholbert** [mozilla/mozmoderator#164]
(Migrating from https://bugzilla.mozilla.org/show_bug.cgi?id=1101981)

In the profile images that Moderator displays next to users' questions, their faces may be squashed, if their profile photo is tall & skinny. (since moderator places it in a square <img> element)

`object-fit: cover` would fix this by preserving the image's aspect ratio when scaling it down.

Please style the profile images (`.selfimage img {...}` and `.askedby img {...}`) with `object-fit: cover`.  (one-liner CSS fix)

Screenshot of what this looks like without the fix, if the person has a tall rectangular profile photo:
  https://bug1101981.bmoattachments.org/attachment.cgi?id=8525658
...and here's a screenshot with `object-fit` applied:
  https://bug1101981.bmoattachments.org/attachment.cgi?id=8525660

**Comment by dholbert** [comment]
(It's been a while since I filed the bugzilla version of this bug, but I just verified that this is indeed still a problem.  I visited https://moderator.mozilla.org/e/monthly-internal-meeting-9-jan-17 , and used devtools to change the "Unknown Profile Image" dino-image to use the wide Moderator logo instead (https://moderator.mozilla.org/static/img/mozillamoderator.af884c85183a.png).  And it gets awkwardly squashed.
